2.4/contrib/bin/run.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# SCL in CentOS/RHEL 7 doesn't support --exec, we need to do it ourselves
+source scl_source enable mongodb24
+set -e
+
+function usage {
+	echo "You must specify following environment variables:"
+	echo "  \$MONGODB_USER"
+	echo "  \$MONGODB_PASSWORD"
+	echo "  \$MONGODB_DB - optional (default database - 'production')"
+	echo "  \$MONGODB_ADMIN_PASSWORD - optional"
+	exit 1
+}
+
+function create_mongodb_users {
+	mongod -f /opt/openshift/etc/mongodb.conf &
+
+	# Check if the MongoDB daemon is up.
+	RC=1
+	while [[ RC -ne 0 ]]; do
+	    echo "=> Waiting for confirmation of MongoDB service startup"
+	    sleep 2
```

Comment:
@bparees you might get different behavior... let say your container was launched on crappy node with corrupted storage or problem in kernel? Then restarting container of different node fixes the problem.

liveness problem capture this, that is true... but then aren't you just delegating shooting container to k8s itself? (and it will try to restart it ;-)

@bparees so what user will do when it gets 'stuck' mongodb container? Should user act and shoot it? For example, if I deploying production app and the mongo gets stuck to whatever livenessProbe is configured (if it is configured) then my app will be broken till I realize that mongodb failed to start and I manually fix it.